.travis.yml

```diff
@@ -58,6 +58,13 @@ before_script:
       export ARTIFACTS_UPLOAD_PATH=${ARTIFACTS_UPLOAD_PATH_PR}
     fi
 
+# *** WARNING ***
+#
+# A single script block must include exactly ONE command.
+# Travis runs a script block without the `-e` flag. Any failures in the middle are simply ignored.
+#
+# Specifying the `-e` flag manually is not recommended. It will stop the execution after the very first failure.
```

Comment:
These lines are a bit long, let's wrap at around 80-100 characters for readability